roles/rsyslog/templates/input_relp.j2

```diff
@@ -49,7 +49,7 @@ if
 {%         if not loop.first %}
   or
 {%         endif %}
-  ($inputname == "{{ item.name }}_{{ loop.index }}" )
+  ( $inputname == "{{ item.name }}" )
```

Comment:
The rsyslog templates are not consistent with whitespace - some templates use `($inputname == something)` - some use `($inputname == something )` and some use `( $inputname == something)` etc.

I would prefer not to change the whitespace here at this time - keep it as `($inputname == something )` - we might revisit this issue in the future to make all of the templates consistent

Reply:
Yeah, I also noticed the whitespace inconistencies in multiple templates... (I was hoping to fix all of them together, but I registed. :)

I thought this particular line is ok since the change is covered by the real bug fix... But if you prefer to keep it, I'll revert the whitespace change.

Reply:
> But if you prefer to keep it, I'll revert the whitespace change.

Let's first agree on a whitespace convention to use for generating rsyslog configs, then we can go back and make them all consistent.

Reply:
Reverted the whitespace change.

Regarding "a whitespace convention to use for generating rsyslog configs", do we have some guidance in the rsyslog team, @sopos? (I'm not planning to solve the problem in this pr, but I'd like to have some inputs for the preparation.)

Looking into the examples in the rsyslog docs, they do not care much about whiltespaces... E.g., in the first example in https://www.rsyslog.com/doc/master/rainerscript/control_structures.html, the first `if` does not have an extra whitespace after `(` and before `)`, but the second `if` does. But at least, it's consistent, of course. If it has one after `(`, then there is another before `)`.
```
if ($msg contains "important") then {
   if ( $.foo != "" ) then set $.foo = $.bar & $.baz;
   action(type="omfile" file="/var/log/important.log" template="outfmt")
}
```

Reply:
no guidance here, no strong opinion neither

Reply:
> no guidance here, no strong opinion neither

Thanks, @sopos.

@richm, is it ok to have one whitespace like this? And open a pr?
`( $inputname == something )`

Reply:
> @richm, is it ok to have one whitespace like this for the control code such as in if and foreach expression? And open a pr?

yes